mitodl/micromasters
# Fixes layout issue with schedule an exam button

#### What are the relevant tickets?
#2854

#### What's this PR do?
This fixes a small issue with the layout of the button in the Pearson Exam card. The button label was getting cut off when there were very long course names. This adds a min-width on a button.

#### How should this be manually tested?
Get into the state where you can schedule a Pearson exam. Make the course name different character lengths. Make sure the button text always is readable as "SCHEDULE AN EXAM".

#### Any background context you want to provide?
This also fixes a couple very minor margin issues with the list item.

#### Screenshots (if appropriate)
**Before**
![image](https://cloud.githubusercontent.com/assets/20047260/23872775/f30619f6-0805-11e7-995b-bdc4dd7a96b8.png)

**After**
![image](https://cloud.githubusercontent.com/assets/20047260/23872756/e0ff0d4e-0805-11e7-87c6-fde01e2932d8.png)
